src/services/ui/src/routes.ts

```diff
@@ -23,3 +23,5 @@ export enum ROUTES {
 export enum FAQ_SECTION {
   SYSTEM = "system",
 }
+
+export const FAQ_TARGET = "faq-terget" as const;
```

Comment:
I think this is a typo. Should be this right
```suggestion
export const FAQ_TARGET = "faq-target" as const;
```